web-admin/src/routes/[organization]/-/upgrade-callback/+page.ts

```diff
@@ -0,0 +1,23 @@
+import type { V1BillingIssue } from "@rilldata/web-admin/client";
+import { checkUserAccess } from "@rilldata/web-admin/features/authentication/checkUserAccess";
+import { getCancelledIssue } from "@rilldata/web-admin/features/billing/issues/getMessageForCancelledIssue";
+import { getPaymentIssues } from "@rilldata/web-admin/features/billing/issues/getMessageForPaymentIssues";
+import { fetchOrganizationBillingIssues } from "@rilldata/web-admin/features/billing/selectors";
+import { error } from "@sveltejs/kit";
+import type { PageLoad } from "./$types";
+
+export const load: PageLoad = async ({ params: { organization } }) => {
+  let issues: V1BillingIssue[] = [];
+  try {
+    issues = await fetchOrganizationBillingIssues(organization);
```

Comment:
I'm wondering if this should happen at the `[organization]` route level (semantically that would make sense), and then this callback should just invalidate the parent load function? But maybe not, because that might lead to excessive invalidation?

Reply:
Lets keep it simple for now.